packages/components/src/templates/next/components/internal/Navbar/MobileNavMenu/MobileNavItemAccordion.tsx

```diff
@@ -27,12 +36,81 @@ const mobileItemStyles = tv({
       true: {
         container: "pb-6",
         chevron: "-rotate-180",
+        menuItemsContainer: "flex flex-col gap-6",
+      },
+      false: {
+        menuItemsContainer: "hidden",
+      },
+    },
+    itemType: {
+      default: {
+        item: "justify-between gap-6",
+      },
+      parentItem: {
+        item: "gap-1",
+        nestedItem: "flex items-center gap-1",
       },
     },
+    withVerticalPadding: {
+      true: {
+        item: "py-1",
+      },
+    },
+  },
+  defaultVariants: {
+    itemType: "default",
   },
 })
 
-const { item, chevron, container, nestedItem, sublist } = mobileItemStyles()
+const { item, chevron, container, nestedItem, sublist, menuItemsContainer } =
+  mobileItemStyles()
+
+const ParentItemLink = ({
+  name,
+  referenceLinkHref,
+  isExternal,
+  LinkComponent,
+}: ParentItemLinkProps) => {
+  // This is a hack to ensure that the rightArrow is always at the end of the last word even on smaller screens
+  const words = name.trim().split(" ")
+  const allButLastWord = words.slice(0, -1).join(" ")
+  const lastWord = words[words.length - 1]
+
+  return (
+    <div
```

Comment:
[nit] won't this mean that clicking on the div **won't** have an effect? if that's the case, wdyt about setting the `link` on the outer item?